measures/measures-schema.yaml

```diff
@@ -300,6 +300,9 @@ definitions:
       procedureCodes:
         type: array
         items: { $ref: #/definitions/codeObject }
+      additionalProcedureCodes:
```

Comment:
I see these aren't present in other fields for `eligibilityOption` but would be nice if it's not too hard to add a description here.